### PR TITLE
Split cars and vans groups

### PR DESCRIPTION
**Pull request by catalinjitea**

### CODE REVIEW

File: notifications/fixtures/companiesgroups.json [hunk 1]
```diff
@@ -3,29 +3,37 @@
     "model": "notifications.companiesgroup",
     "pk": 1,
     "fields": {
-      "title": "Cars/Vans",
-      "code": "cars-vans"
+      "title": "Cars",
```
Comment (dianaboiangiu) [inline notifications/fixtures/companiesgroups.json]: Better set id for cars and vans  5-6 than changing the ids for the others

File: notifications/models.py [hunk 2]
```diff
@@ -46,10 +46,11 @@ def __unicode__(self):
 
 class CompaniesGroup(models.Model):
     """ Base class for the 3 groups of companies:
-        1 - Cars/Vans
-        2 - F-gases EU
-        3 - F-gases NONEU
-        4 - ODS
+        1 - Cars
```
Comment (dianaboiangiu) [inline notifications/models.py]: modify here for the ids too

**Review by dianaboiangiu — CHANGES_REQUESTED** [review]

File: notifications/models.py (hunk 2 above)
Comment (catalinjitea) [reply notifications/models.py]: Done.

File: notifications/fixtures/companiesgroups.json (hunk 1 above)
Comment (catalinjitea) [reply notifications/fixtures/companiesgroups.json]: Done.

**Pull request merged**